<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ventana</class>
 <widget class="QWidget" name="Ventana">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>525</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>352</width>
    <height>525</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Ink Free</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Ventana</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="Boton" name="pbMiDentista">
     <property name="font">
      <font>
       <family>Lucida Sans</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Mi Dentista</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="Boton" name="pbVisitas">
     <property name="font">
      <font>
       <family>Lucida Sans</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Visitas</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="Boton" name="pbTecnicasHigiene">
     <property name="font">
      <font>
       <family>Lucida Sans</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TÃ©cnicas de Higiene</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="Boton" name="pbMiBoca">
     <property name="font">
      <font>
       <family>Lucida Sans</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Mi Boca</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="Boton" name="pbHoraCepillarse">
     <property name="font">
      <font>
       <family>Lucida Sans</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Hora de Cepillarse</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Boton</class>
   <extends>QPushButton</extends>
   <header location="global">boton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
